frontend/src/services/SmallestPriceService.js

```diff
@@ -0,0 +1,17 @@
+export default class SmallestPriceService {
```

Comment:
ich glaube, da hatten wir aneinander vorbeigeredet, ich dachte, dass du die Funktion in den `ProductService` integrierst. Der `SmallestPriceService` hätte ja vermutlich immer nur diese eine Funktion...

Reply:
Ja das hatte ich anders verstanden. Macht meiner Meinung aber auch mit dem ProductService mehr Sinn natürlich.